util/pastPerformance/testReleasesPerformance

```diff
@@ -47,6 +47,14 @@ fi
 export CHPL_HOME_ORIG=$CHPL_HOME
 export CHPL_TEST_UTIL_DIR=$CHPL_HOME_ORIG/util
 
+if [[ -z $CHPL_TEST_VENV_DIR && ! -z $CHPL_HOST_PLATFORM ]]; then
+    export CHPL_TEST_VENV_DIR=$CHPL_HOME/third-party/chpl-venv/install/$CHPL_HOST_PLATFORM/chpl-virtualenv
+fi
+if [[ ! -d $CHPL_TEST_VENV_DIR ]]; then
+    echo "ERROR: incorrect or missing CHPL_TEST_VENV_DIR or CHPL_HOST_PLATFORM"
```

Comment:
Maybe echo the directory that it was testing for as well to help with debugging?